build_utils/docker_prepare_manylinux2014_aarch64.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -e
+
+yum install -y epel-release
+yum install -y pcre pcre-devel openssl-devel wget libtool snappy-devel gettext gettext-devel po4a nasm meson cmake3 ninja-build
+
+# On aarch64 machine, yum pull meson==0.47.0 and project requires meson >= 0.49.0 so installing it from source code
+wget https://github.com/mesonbuild/meson/releases/download/0.49.0/meson-0.49.0.tar.gz
+tar -xvzf meson-0.49.0.tar.gz
+cd meson-0.49.0
+export PATH=/opt/python/cp38-cp38/bin/:$PATH
+python setup.py install
+cd ..
+rm -rf meson-0.49.0 meson-0.49.0.tar.gz
```

Comment:
why this instead of `pip install`? Why this old version instead of the newest one? 
```suggestion
export PATH=/opt/python/cp38-cp38/bin/:$PATH
pip install meson==0.49.0
```

Reply:
Installing meson through pip. latest meson is thronging build error. Lets use  meson==0.49.0 for now.